app/lib/efile/nj/nj1040_calculator.rb

```diff
@@ -109,6 +110,29 @@ def calculate_tax_liability_without_deduction
         ((income * rate) - subtraction).round(2)
       end
 
+      def calculate_use_tax(nj_gross_income)
+        case nj_gross_income
+        when 0..15_000
+          14
+        when 15_000..30_000
+          44
+        when 30_000..50_000
+          64
+        when 50_000..75_000
+          84
+        when 75_000..100_000
+          106
+        when 100_000..150_000
+          134
+        when 150_000..200_000
+          170
+        when 200_000..Float::INFINITY
+          [0.000852 * nj_gross_income, 494].min.round
+        else
```

Comment:
[DUST] Do we ever expect this to be 0? Looking at the tests, it seems like this should always have a positive value if we're autocalculating for them. Is there an alternative way to capture this error state?

Reply:
maybe the first case could be `when -Float::INFINITY..15_000` instead of `when 0..15_000`

and then the `when 200_000..Float::INFINITY` could become the `else` (with a comment of `# above 200_000`) and then the current `else` case could just be removed?

Reply:
The only case we're solving for here is if income is negative, right? Since that should never happen, I'd rather have the explicit `else` to just set this to 0. Even though it should never happen. But seems better than capturing it in the first case and assigning $14 of use tax

Reply:
throwing an error in the "should never happen" case instead of returning `0` then?

Reply:
@mluedke2 idk... it just feels weird to throw an error in the app when I don't know enough about how it would get handled for the user experience. Something just feels weird about throwing an error. Maybe it doesn't matter since it should never happen, but it just makes me worried I'm missing something and then a user will end up with an error.

What do you think is problematic with the return 0 case?

Reply:
i don't think it's blocking for this PR, but i think it's better to fail loudly and early-- if in the weird case we *are* missing something, i'd much rather give an error that will hopefully trigger our monitoring etc than quietly give the user a wrong tax return. the problem with returning `0` for a negative value is that it turns an invalid input into a valid output that can look normal and cause problems downstream.

Reply:
@embarnard That's interesting! In what circumstances could that happen?

Reply:
also interested how that might happen -- regardless, that leads me to believe that `when -Float::INFINITY..15_000` might be the route to go after all (and then we could remove the `else` since our cases would cover all possibilities)

Reply:
Yes I think you're right @mluedke2, for some reason I thought if AGI was negative it should return 0 which the else case catches but found this comment from @mrotondo which leads be to believe it should instead use the `when -Float::INFINITY..15_000` case:
```
heads up/reminder for anyone writing calculator code, especially anyone creating one of our many tables that take AGI and match it against a bunch of ranges: AGI can be negative, so anytime we have a chart where the lowest band starts at 0, we need to either use [0, agi].max or replace the 0's in the range with -Float::INFINITY (the latter is what we've tended to do in the codebase)
```
I'm realizing I need to change this for NC sales use tax calculation as well

Reply:
Brought this up today with Taxation, their guidance is to return 14$ for negative AGI.